styles/navigation/navbar/navbar.twig

```diff
@@ -0,0 +1,9 @@
+<ul class="navbar">
+  {%for item in items%}
+    <li class="navbar__item">
+      <a href="{{ item.url }}" class="navbar__link{% if item.active %} is-active{% endif %}">
```

Comment:
1. Ok
2. Ei kai for-looppia tarvita? Jotain tällasta siis mietin (pseudo-koodia):
```
set classes = [navbar__link]
if item.active
  set classes = classes|merge([is-active])
endif
```

Kaikki käy omasta puolesta. Tuli vaan mieleen jos tollanen ois hyvä.